automation/check-patch.e2e-upgrade-k8s.sh

```
@@ -22,6 +22,7 @@ teardown() {
     make cluster-down
     # Don't fail if there is no logs
     cp -r ${E2E_LOGS}/operator/* ${ARTIFACTS} || true
+    cp -r ${E2E_LOGS}/handler/* ${ARTIFACTS} || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)